webpack/watchConfig.js

```diff
@@ -1,6 +1,8 @@
 module.exports = {
   watch: true,
   watchOptions: {
+    ignored: /.cpanm/,
     ignored: /node_modules/,
+    ignored: /perl_modules/,
```

Comment:
Same here, Webpack shouldn't look in these unless you tell it to, so no need to ignore